lib/hammer_cli_foreman_remote_execution/interface_extensions.rb

```diff
@@ -0,0 +1,13 @@
+require 'hammer_cli_foreman/interface'
+
+module HammerCLIForemanRemoteExecution
+  class InterfaceExtensions < ::HammerCLI::CommandExtensions
+    output do |definition|
+      definition.append do
+        field :execution, _('Execution')
+      end
+    end
+  end
+
+  ::HammerCLIForeman::Interface::InfoCommand.extend_with(InterfaceExtensions.new)
+end
```

Comment:
Missing newline at the end of the file :)

Reply:
added, thanks!